src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Lottery {
+    private Ball ball = new Ball();
+
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public String getRandomBall() {
+        ball.setColor(colorSupplier.getRandomColor());
+        ball.setNumber(colorSupplier.getRmInt(100));
```

Comment:
lets use Random object directly from here, ColorSupplier is meant to randomize colors not numbers
also 100 is a magic number ✨  use constants